pkg/service/service.go

```diff
@@ -162,8 +167,13 @@ func NewService(monitorNotify monitorNotify) *Service {
 		monitorNotify:   monitorNotify,
 		healthServer:    localHealthServer,
 		lbmap:           lbmap.New(maglev, maglevTableSize),
+		l7lbSvcs:        map[string]uint16{},
 	}
 	svc.lastUpdatedTs.Store(time.Now())
+
+	// XXX: Hard-coded test data
+	svc.l7lbSvcs["cilium-test/echo-other-node"] = uint16(9090)
```

Comment:
Leftovers from debugging?

Reply:
Yes, removed in later commits, will clean up on the second round of reviews.